model/registry/user.yaml

```diff
@@ -0,0 +1,50 @@
+groups:
+  - id: registry.user
+    prefix: user
+    type: attribute_group
+    brief: "Describes information about the user that is relevant to the event."
+    attributes:
+      - id: domain
```

Comment:
I wonder if `issuer` is not a better name? This is also aligned with the use in JWT.

Reply:
We have an [agreement](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#merging-existing-ecs-conventions) to not override ecs naming unless there are significant reasons for it.
Personally for me both `domain` and `issuer` are similar, where `domain` is more clearly defines what field represents - url

Reply:
Since we're renaming the namespace anyway, let's revisit the naming.

Domain is a very broad term `issuer` would be much more specific and descriptive.

Reply:
Why we renaming the namespace anyway?

Reply:
We are not renaming enduser to user. Enduser is getting deprecated and user should be used in places where enduser was used but not only.

Let's discuss this particular field separately. I will create an issue.